test/common/wpt.js

```diff
@@ -6,9 +6,92 @@ const fs = require('fs');
 const fsPromises = fs.promises;
 const path = require('path');
 const events = require('events');
+const os = require('os');
 const { inspect } = require('util');
 const { Worker } = require('worker_threads');
 
+function getBrowserProperties() {
+  const release = !!process.release.sourceUrl;
+  const browser = {
+    browser_channel: release ? 'stable' : 'experimental',
+    browser_version: process.version,
+  };
+
+  if (release) {
+    browser.browser_version = process.version;
+  } else if (process.env.NODE_REVISION) {
+    browser.browser_version = process.version.replace('pre', process.env.NODE_REVISION);
+  }
+
+  return browser;
+}
+
+function getOs() {
```

Comment:
Can you add a comment about where you found out that Darwin == win? It would be helpful for future reference.